tests/wiseagents/agents/test_phased_coordinator.py

```diff
@@ -81,6 +81,14 @@ def test_phased_coordinator():
                                    "Coordinator")
         cond.wait()
 
-        print(f"registered agents= {WiseAgentRegistry.get_agents()}")
+        print(f"registered agents= {WiseAgentRegistry.fetch_agents_descriptions_dict()}")
         for message in WiseAgentRegistry.get_or_create_context('default').message_trace:
-            print(f'{message.sender} : {message.message} ')
\ No newline at end of file
+            print(f'{message.sender} : {message.message} ')
+
+    #stop agents
+    client_agent1.stopAgent()
```

Comment:
We might want to add `try/finally` in the `with` block and move the stopping of the agents to the `finally` section to make sure the agents are still stopped even if something goes wrong in the test.

Reply:
Same with the other tests too, we should make sure that the agents will be stopped even if an exception occurs during the test.